src/components/lists/lists.css

```diff
@@ -0,0 +1,41 @@
+.lists__head {
+    display: flex;
+    justify-content: space-between;
+    gap: 2rem;
+    padding: 8px 0;
+
+}
+
+.lists__menu {
+    list-style: none;
+    padding: 0 1rem;
+	border-radius: 8px;
+    border: 1px solid var(--COLOR-PRIMARY);
+}
+
+.lists__spaced {
+    gap: 3px;
+}
+.lists--stacked{
+    gap:0;
+}
+.lists__head {
+	font-weight: 500;
+	font-size: 2.2rem;
+
```

Comment:
avoid these line gaps and give line gap between classes

Reply:
Thanks for reviewing and mentioning changes, will update it all.